src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -1841,5 +1855,202 @@ string[] linkingTargetFields
                 configValidator.ValidatePermissionsInConfig(runtimeConfig);
             }
         }
+
+        /// <summary>
+        /// Test to validate that the rest methods are correctly configured for entities in the config.
+        /// Rest methods can only be configured for stored procedures as an array of valid REST operations. 
+        /// </summary>
+        /// <param name="sourceType">The source type of the entity.</param>
+        /// <param name="methods">Value of the rest methods property configured for the entity.</param>
+        /// <param name="exceptionExpected">Boolean value representing whether an exception is expected or not.</param>
+        /// <param name="expectedErrorMessage">Expected error message when an exception is expected for the test run.</param>
+        [DataTestMethod]
+        /*[DataRow(EntitySourceType.Table, "[\"get\"]", true,
+            $"The rest property '{Entity.PROPERTY_METHODS}' is present for entity: HybridEntity of type: Table, but is only valid for type: StoredProcedure.",
+            DisplayName = "Rest methods specified for non-storedprocedure entity fail config validation.")]*/
```

Comment:
commented out code can be removed.

Reply:
Going to use ignore on this test right now, because with Aaron's PR, this will pass even for tables/views - when it shouldn't. Once we have that change in, we can un-ignore this.

Reply:
Yeah here: https://github.com/Azure/data-api-builder/issues/1570